InternetHospital.WebApi/Controllers/SignupController.cs

```diff
@@ -49,7 +63,7 @@ public async Task<ActionResult> Register([FromBody] UserRegistrationModel vm)
                             return BadRequest("Error during patient registration");
                         }
                         callbackUrl = await GenerateConfirmationLink(user);
-                    }
+                    } 
                     else if (vm.Role == "Doctor" || vm.Role == "doctor")
```

Comment:
I don't think, that there is need in checking both "doctor" and "Doctor" as well as patient. The name of role is const.